docs/docker.md

```diff
@@ -2,15 +2,33 @@
 
 ## Overview
 
-Docker can automate much of the configuration for you.
+Docker packages software into virtual containers so it can run on any Linux, Windows, or macOS computer. It can automate much of the configuration for you.
 
-
-## Using docker
+## Using Docker
 
 ### Prerequisites
 
-* Recent versions of `docker` and `docker-compose`.
-* A database dump of the full AGAGD schema, stored at `schema.sql`. (in the root directory of the repository)
+- Recent versions of `docker` and `docker-compose`.
+- A database dump of the full AGAGD schema, stored at `schema.sql`. (in the root directory of the repository)
+
+### [Installation](https://docs.docker.com/engine/install)
+
+#### Mac & Windows
+
+Installing the [Docker Desktop](https://www.docker.com/products/docker-desktop) application is highly recommended.
+
+#### Linux
+
+The Docker Engine is available on a variety of Linux platforms:
+
```

Comment:
This should be more specific to being an installation step [Docker Compose Install](https://docs.docker.com/compose/install/). I also think you are missing the docker-compose installation. Also, would probably apply